src/lib/CLValue/Result.ts

```diff
@@ -143,13 +143,21 @@ export class CLResult<T extends CLType, E extends CLType> extends CLValue {
     this.innerOk = ok;
     this.innerErr = err;
   }
+
   /**
    * Returns Result from ts-result based on stored value
    */
   value(): Result<CLValue, CLValue> {
     return this.data;
   }
 
+  /**
+   * Returns JSON representation. If None null will be returned.
+   */
+  toJSON(): any {
+    return this.isError() ? this.data.err : this.data.unwrap().toJSON();
```

Comment:
Do you not need to put a `.toJSON()` on the `this.data.err` because it is a `CLValue` (from https://github.com/casper-ecosystem/casper-js-sdk/blob/4c16adcc5f9125ba8c8d377c091f75abb80e6298/src/lib/CLValue/Result.ts#L136)
Otherwise it won't be formatted properly, unless I'm misunderstanding.

Reply:
in fact in both cases its CLValue, so the check was unnecessary. Thx :+1: